From 0d96f010e771f150a704edb6c651c62db6cbd227 Mon Sep 17 00:00:00 2001
From: Prabhakar Kushwaha <prabhakar@freescale.com>
Date: Tue, 19 Jan 2016 12:02:49 +0530
Subject: [PATCH 49/74] driver: net: fsl-mc: Update print to reflect correct
 string

Update printf with dpbp_exit to match with previous function call.

Signed-off-by: Itai Katz <itai.katz@nxp.com>
Signed-off-by: Prabhakar Kushwaha <prabhakar.kushwaha@nxp.com>
---
 drivers/net/fsl-mc/mc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/fsl-mc/mc.c b/drivers/net/fsl-mc/mc.c
index 9462b5b..29124f3 100644
--- a/drivers/net/fsl-mc/mc.c
+++ b/drivers/net/fsl-mc/mc.c
@@ -1160,7 +1160,7 @@ int fsl_mc_ldpaa_exit(bd_t *bd)
 
 	err = dpbp_exit();
 	if (err < 0) {
-		printf("dpni_exit() failed: %d\n", err);
+		printf("dpbp_exit() failed: %d\n", err);
 		goto err;
 	}
 
-- 
2.1.0.27.g96db324

